incuna_auth/middleware/utils.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+# Python 2/3 compatibility hackery
+try:
+    unicode
+except NameError:
+    unicode = str
```

Comment:
Can you please use `django.utils.six` for this instead for the sake of uniformity?